could not find PostgreSQL (missing: PostgreSQL_INCLUDE_DIR)

Hi! I'm trying to install Nominatim on MacOS Sierra (10.12.3).

I've stumbled upon the following error when running the `$ cmake ..` command from within the `build` directory:

```
CMake Error at CMakeLists.txt:37 (add_subdirectory):
  The source directory

    /Users/adler/Nominatim/osm2pgsql

  does not contain a CMakeLists.txt file.


CMake Error at /usr/local/Cellar/cmake/3.8.0-rc1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find PostgreSQL (missing: PostgreSQL_INCLUDE_DIR)
```

For my current postgres installation, I'm using the [Postgres app](https://postgresapp.com/) for OS X. Hope you can help!

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,6 +34,12 @@ add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
 set(BUILD_TESTS off CACHE BOOL "Build test suite" FORCE)
 set(WITH_LUA off CACHE BOOL "Build with lua support" FORCE)
+
+if (NOT EXISTS "${CMAKE_SOURCE_DIR}/osm2pgsql/CMakeLists.txt")
+    message(FATAL_ERROR "The osm2pgsql directory is empty.\
+    Did you forget to check out Nominatim recursively?\
+    \nTry updating submodules with: git submodules update --init")
+endif()
 add_subdirectory(osm2pgsql)
 
 find_package(Threads REQUIRED)

